src/renderers/shared/shared/event/BrowserEventConstants.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule BrowserEventConstants
+ */
+
+'use strict';
+
+var getVendorPrefixedEventName = require('getVendorPrefixedEventName');
+
+export type PropagationPhases = 'bubbled' | 'captured';
```

Comment:
Rollover from [EventConstants]https://github.com/facebook/react/blob/master/src/renderers/shared/shared/event/EventConstants.js#L14). Happy to remove though.